recipes/baked_beans.json

```diff
@@ -0,0 +1,18 @@
+{
+    "recipe_name": "Baked Beans",
+    "serves": "1",
+    "prep_time": "2 mins",
```

Comment:
A strong guy like me would only take 1 minute to open a can of baked beans so lets make it 1 min.